src/sql/workbench/browser/editor/resourceViewer/resourceViewerInput.ts

```diff
@@ -47,7 +52,15 @@ export class ResourceViewerInput extends EditorInput {
 			sortable: false
 		});
 		this._register(this._actionsColumn.onClick(args => {
```

Comment:
not interested in showing this also when context menu is requested?

Reply:
Added 😄